src/js/components/Drop/DropContainer.js

```diff
@@ -166,32 +166,20 @@ const DropContainer = forwardRef(
           let bottom;
           let maxHeight = containerRect.height;
           if (align.top) {
-            if (align.top === 'top') {
-              ({ top } = targetRect);
-            } else {
-              top = targetRect.bottom;
-            }
-
-            // Calculate visible area underneath the control w.r.t window height
-            const percentVisibleAreaBelow =
-              100 - (targetRect.bottom / windowHeight) * 100;
+            const alignToTop = align.top === 'top';
+            const switchTop = !alignToTop ? targetRect.top : targetRect.bottom;
+            top = alignToTop ? targetRect.top : targetRect.bottom;
 
-            // Check whether it is within 20% from bottom of the window or
-            // visible area to flip the control
-            // DropContainer doesn't fit well within visible area when
-            // percentVisibleAreaBelow value<=20%
-            // There is enough space from DropContainer to bottom of the window
-            // when percentVisibleAreaBelow>20%.
-
-            if (windowHeight === top || percentVisibleAreaBelow <= 20) {
+            if (
```

Comment:
can we add a new comment for this section 
the old comment was clear but since the behavior changed just for future reference